amp/WEB-INF/src/org/digijava/module/aim/util/LocationSkeletonBuilder.java

```diff
@@ -0,0 +1,88 @@
+package org.digijava.module.aim.util;
+
+public class LocationSkeletonBuilder {
+    private Long id;
```

Comment:
Usually, the builder should contain the insatnce of the target class and the method could be:
withId(Long id) {
    skeleton.setId(id);
}

and the instance can be get with build() method
ActivityBuilder could be used asa model.